components/admin_console/system_user_detail/system_user_detail.jsx

```diff
@@ -59,9 +62,34 @@ export default class SystemUserDetail extends React.Component {
             user: {
                 email: this.props.user.email,
             },
+            addTeamOpen: false,
+            refreshTeams: true,
         };
     }
 
+    teamsData = (teams) => {
```

Comment:
I'm wondering if we should maybe call this something like `setTeamsData` so that it reads more like an action than a property?

Reply:
Good suggestion.